packages/orchestrator-ui-components/src/api/index.ts

```diff
@@ -24,9 +24,9 @@ import { ProductDefinition } from '@/types';
 
 import { getAxiosInstance } from './axios';
 
-const CIM_FORMS_ENDPOINT = 'surf/cim/forms/';
 const PROCESS_ENDPOINT = 'processes/';
 const PRODUCTS_ENDPOINT = 'products/';
+const FORMS_ENDPOINT = 'surf/forms/';
```

Comment:
surf specific things should be placed in the surf specific app repository

Reply:
This is for the CreateForm endpoint that is also in wfo-ui. I agreed with Ruben to not delete it for now.